env-default/bin/detect

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+# always successful
```

Comment:
instead of having it here, we could also port the whole buildpack-builder logic to a global "runway-defaults" buildpack, and do actual detection of stuff - maybe a bit cleaner, but maybe also more confusing because the source of the env vars is further away.

with this, it's at least just all in one place